index.html

```diff
@@ -8,34 +8,204 @@
+                    <div class="lang3"><img src="image/Tag.svg" alt="HTML Tag"></div>
+                </div> 
+                <div class="nav-button">
+                    <a href="https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png"><img src="image/Enabled.svg" alt="See Project button"></a> 
+                </div>    
+            </div>
+        </section>
+
+        <section>
+        <div class="container_6"> 
+                <div class="header_6">
+                    <img src="image/Primary Text.svg" alt="Javascript">
+                </div>    
+            <p class="content_6">Hello I’m a software developer! I can help you build a product , feature or website
+                     Look through some of my work and experience! If you like what you see and have a 
+                     project you need coded, don’t hestiate to contact me.</p>
+            <div class="connect"><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img src="image/Social Media.svg" alt="Let's Connect"></a></div>
+            <div class="social">
+                <ul>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="twitter" src="image/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="linkedin" src="image/Linkedin icon.svg" alt="Linkedin"></a></li>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="medium" src="image/medium.svg" alt="Medium"></a></li>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="github" src="image/github.svg" alt="GitHub"></a></li>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="angellist" src="image/angellist.svg" alt="Angellist"></a></li>
+                </ul>
+            </div>
+            <div class="nav-button">
+                <a href="https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png"><img src="image/Resume.svg" alt="Get my Resume"></a> 
+            </div> 
+            <h2 class="nav_language">Languages</h2>
+            <div class="language">
+                <div>
+                    <img src="image/Frame 9.svg" alt="Javascript">
+                </div>
+                <div>
+                    <img src="image/Frame 10.svg" alt="Javascript">
+                </div>
+                <div>
+                    <img src="image/Frame 11.svg" alt="Javascript">
+                </div>
```

Comment:
- [ ] Here also, it's better to use the unordered list `ul and li` instead of `div.` for semantic reasons and user experience (UX).
- [ ] Kindly stick to the font used in your chosen template and add the missed icon in the about section.
- [ ] Please remove the image used for the technologies in the about section and use the proper image icon and the technologies as in the template you chose.

***Template design***
![Screenshot from 2022-12-03 11-26-55](https://user-images.githubusercontent.com/65129228/205435229-405d697d-0fd5-4342-8faf-6211bd90976b.png)

***Your design***
![Screenshot from 2022-12-03 12-15-41](https://user-images.githubusercontent.com/65129228/205435313-eaf2a98c-7b5b-4996-919f-256e1aa10cec.png)